pkg/nitrite_error/nitrite_error.go

```diff
@@ -0,0 +1,30 @@
+package nitrite_error
```

Comment:
Having said that, I think we can depart from having sentinel errors altogether. None of these errors are retriable, so it really doesn't matter what the error is in terms of how to handle it.
My real suggestion is to get rid of these typed errors altogether and instead of returning them, create a return a new generic error with the desired message. In your tests you can test that the error has the desired string. At least this is Dave's current thinking on how to deal with errors based on
https://dave.cheney.net/2014/12/24/inspecting-errors
https://dave.cheney.net/2016/04/07/constant-errors
https://dave.cheney.net/2016/04/27/dont-just-check-errors-handle-them-gracefully

Reply:
Ahhh I definitely read that article wrong and implemented the way he said not to do it... Changed to constant strings which are used to create generic errors (wrapping existing errors where appropriate)